Rules.md

```diff
@@ -203,15 +203,16 @@ A battle has one or more rounds. Every round has following parts:
 1. **Attacker decides to retreat or has no healthy units left:** All units move back to the last
    touched unoccupied, sea or owned region.
 1. **Defender decides to retreat or has no healthy units left:** All units (including wounded units)
-   move to an adjacent land region that is unoccupied or owned by the defender.
+   move to a single adjacent land region that is unoccupied or owned by the defender. Exception:
+   units can't retreat to the region where the attacker's move token was resolved.
```

Comment:
Ok, true, my apologies, then I'd suggest:
"can't retreat to the region where the attacker's move started."